prow/jobs/nats-manager/nats-manager-generic.yaml

```diff
@@ -13,7 +13,7 @@ presubmits: # runs on PRs
         prow.k8s.io/pubsub.runID: "pull-nats-manager-build"
         prow.k8s.io/pubsub.topic: "prowjobs"
         preset-sa-kyma-push-images: "true"
-      run_if_changed: '^Dockerfile|\.go$|^go.(sum|mod)'
+      run_if_changed: '^.env$|^hack/ci/*/.*$|^pkg/*/.*$|^internal/*/.*$|^cmd/*/.*$|^resources/*/.*$|^api/*/.*$|^Dockerfile$|^(go.mod|go.sum)$|^*/(.*.go|Makefile|.*.sh)|^PROJECT$|^config/*/.*$|^docs/*/.*$'
```

Comment:
Why not run it if anything changed?

Also shouldn't `/*/` change to `/.*/`? or did I miss something?

Reply:
made post-main jobs to always run, and pre-main to skip only for docs.